src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java

```diff
@@ -717,13 +745,13 @@ public static View createBuildingView(ColonyView colony, BlockPos id, ByteBuf bu
                     if (c.getName().endsWith("$View"))
                     {
                         Constructor<?> constructor = c.getDeclaredConstructor(ColonyView.class, BlockPos.class);
-                        view = (View)constructor.newInstance(colony, id);
+                        view = (View) constructor.newInstance(colony, id);
                         break;
                     }
                 }
             }
-        }
-        catch (NoSuchMethodException | IllegalAccessException | InvocationTargetException | InstantiationException exception)
+        } catch (NoSuchMethodException | IllegalAccessException | InvocationTargetException
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this "catch" keyword to a new dedicated line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ARightCurlyBraceDifferentLineAsNextBlockCheck)